qiita_pet/support_files/doc/source/checklist-for-ebi-ena-submission.rst

```diff
@@ -0,0 +1,79 @@
+Checklist to send data to EBI-ENA
+=================================
+
+For each preparation that needs to be uploaded to EBI-ENA we will check:
+
+  1. Data processing
+
+    a. Make sure there is raw data linked - no BIOMs allowed
+    b. The data is processed and the owner confirms the data is correct:
+
+      1. For target gene: data is demultiplexed (review split_library_log to make sure it seems fine) and there is at least a close reference (GG for 16S, Silva for 18S, UNITE for ITS) or trim/deblur artifacts. Trimming should be done with 90, 100 and 150 base pairs (prefered).
+      2. For shotgun: data is uploaded via per_sample_FASTQ and processed using Shogun/utree
```

Comment:
Perhaps worth adding a reminder that the uploaded data should have the host dna removed, maybe link to the tutorials on how to do this.